client/scss/components/_figure.scss

```diff
@@ -83,6 +83,7 @@
   .figure__image {
     max-height: 70vh;
     width: auto;
+    max-width: 100%;
```

Comment:
Not related to macros – but keeping the hero image centered/making sure we don't get horizontal scroll bars on narrower screens.